app/assets/stylesheets/header.css.scss

```diff
@@ -72,12 +72,11 @@ li:hover ul {
 
 
 .logo {
-  border-style: outset;
-  border-width: thick;
-  height: 200px;
+  box-shadow: -10px 10px 30px 0 rgba(0, 0, 0, 0.75);
```

Comment:
Color literals like `rgba(0, 0, 0, 0.75)` should only be used in variable declarations; they should be referred to via variable everywhere else.